src/view-model.ts

```diff
@@ -70,6 +73,8 @@ module ViewModel {
       id = id || Utils.randomString();
       var note = new Note();
       note.id = id;
+      note.title( "Title here" );

```

Comment:
It seems that `Title here` is not a real value, could you find another possibility? for example to renter `Title here` when the value is null or empty.

BTW, extra spaces:
```
note.title( "Title here" );
           ^            ^
```